metadata-ingestion/tests/unit/utilities/test_utilities.py

```diff
@@ -1,8 +1,39 @@
 import doctest
+from typing import List
 
+from datahub.sql_parsing.schema_resolver import SchemaResolver
+from datahub.sql_parsing.sqlglot_lineage import SqlParsingDebugInfo, sqlglot_lineage
 from datahub.utilities.delayed_iter import delayed_iter
 from datahub.utilities.is_pytest import is_pytest_running
-from datahub.utilities.sql_parser import SqlLineageSQLParser
+from datahub.utilities.urns.dataset_urn import DatasetUrn
+
+
+class SqlglotSQLParser:
+    def __init__(self, sql_query: str, platform: str = "bigquery") -> None:
+        self.result = sqlglot_lineage(sql_query, SchemaResolver(platform=platform))
+
+    def get_tables(self) -> List[str]:
+        ans = []
+        for urn in self.result.in_tables:
+            table_ref = DatasetUrn.from_string(urn)
+            ans.append(str(table_ref.name))
+        return ans
+
+    def get_columns(self) -> List[str]:
+        ans = set()
+        for col_info in self.result.column_lineage or []:
+            for col_ref in col_info.upstreams:
+                ans.add(col_ref.column)
+        return list(ans)
+
+    def get_downstream_columns(self) -> List[str]:
```

Comment:
it seems like the new `get_downstream_columns` is similar in functionality to the original `get_columns`? or am I mistaken on that?

overall I'm a bit confused why you added that extra method, when we previously didn't have anything like it?

Reply:
hi @hsheth2  get_downstream_columns it was not there previously i have added this to check what we are getting 
in downstream columns as our new parser we get both upstream and downstream where we wont get in sqllineage,

One more get_columns of sqllinage parser is not being used anywhere in source code its only used testcases.